frontend/.env.example

```diff
@@ -16,4 +16,7 @@ REACT_APP_API="http://localhost:4000"
 # Fortmatic wallet - not used yet
 REACT_APP_FORTMATIC_KEY=""
 
-REACT_APP_PAYMENT_TOKEN_NAME="ETH"
\ No newline at end of file
+REACT_APP_PAYMENT_TOKEN_NAME="ETH"
+
+
+INFURA_PROJECT_ID="9aa3d95b3bc440fa88ea12eaa4456161"
```

Comment:
Needs to be prefixed with `REACT_APP_` or it won't be read by CRA.

Reply:
added